scipy/optimize/optimize.py

```diff
@@ -425,8 +424,26 @@ def _minimize_neldermead(func, x0, args=(), callback=None,
         ``initial_simplex[j,:]`` should contain the coordinates of
         the j-th vertex of the ``N+1`` vertices in the simplex, where
         ``N`` is the dimension.
-
+    xatol : float, optional
+        Absolute error in xopt between iterations that is acceptable for
+        convergence.
+    fatol : number, optional
+        Absolute error in func(xopt) between iterations that is acceptable for
+        convergence.
     """
+    if 'xtol' in unknown_options:
+        xatol = unknown_options['xtol']
+        warnings.warn("xtol is deprecated for _minimize_neldermead,"
+                      " use xatol instead.",
+                      DeprecationWarning)
+        unknown_options.pop('xtol')
+    if 'ftol' in unknown_options:
+        fatol = unknown_options['ftol']
+        unknown_options.pop('ftol')
+        warnings.warn("ftol is deprecated for _minimize_neldermead,"
+                      " use fatol instead.",
+                      DeprecationWarning)
+
```

Comment:
warnings.warn should be fine.

It might be nicer if `options=dict(ftol=foo, fatol=bar)` implied `fatol=bar`, but this will probably not really matter much.